deploy/crds/neutron.openstack.org_v1_ovncontroller_cr.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: neutron.openstack.org/v1
+kind: OVNController
+metadata:
+  name: ovncontroller
+  namespace: openstack
+spec:
+  ovnControllerImage: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7d6ba1bcc4f403733e44cd2a008dafe1887501f7a3c74084566a4f4d1e46250d
```

Comment:
shouldn't this be tripleO image?

Reply:
> shouldn't this be tripleO image?

eventually we'd like OCP and OSP to use the same versions. But for now if we want them to match choosing the OCP version here is probably our only option. If the versions don't need to match (because we aren't testing any advanced bridging or anything) then using the RDO image would be better though